test/spec/test_helper/load.bash

```diff
@@ -0,0 +1,122 @@
+# Copyright 2018 Google LLC. This software is provided as-is,
```

Comment:
Since we don't have defined boilerplate for `.bash` files I think `verify_boilerplate.py` missed this boilerplate. Should this be the standard Google Apache 2 license header?

Reply:
Yes, good catch.

Reply:
Fixed in a3cea47...7e56437 HEAD -> test_structure (forced update)

Also added boilerplate for bats and bash files to catch this next time.